src/runtime/runtime.js

```diff
@@ -876,9 +891,9 @@ function createPublicRuntime(runtime) {
     saveSubscription: runtime.saveSubscription.bind(runtime),
     createButton: runtime.createButton.bind(runtime),
     attachButton: runtime.attachButton.bind(runtime),
+    getPropensityModule: runtime.getPropensityModule.bind(runtime),
     attachSmartButton: runtime.attachSmartButton.bind(runtime),
```

Comment:
any reason to change the order?

Reply:
Not really, it just happened during merge.  Plus it seemed like the smart button came in after it so it should be last.  Doesn't really matter though